runtime/src/main/java/org/corfudb/runtime/view/AbstractReplicationView.java

```diff
@@ -1,129 +0,0 @@
-package org.corfudb.runtime.view;
```

Comment:
I understand we are focusing this branch on quorum replication only. But why not retain the abstractions and interfaces that allow flexible replication schemes in the future?